src/main/java/com/sweetwith/dailynote/domain/file/File.java

```diff
@@ -0,0 +1,26 @@
+package com.sweetwith.dailynote.domain.file;
+
+import com.sweetwith.dailynote.domain.BaseTimeEntity;
+import lombok.*;
+
+import javax.persistence.*;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@Builder
+@Getter
+public class File extends BaseTimeEntity
+{
+    @Id
+    private Long fileId;
+
+    @Column(nullable = false) private String fileName;
+    @Column(nullable = false) private String filePath;
+    @Column(nullable = false) private String fileType;
+    @Column(nullable = false) private String fileAuthor;
+    @Column(nullable = false) private Long fileSize;
+    @Column(nullable = false) private Long postId;
```

Comment:
How about binding post and user with Foreign key with ManytoOne?
https://dublin-java.tistory.com/51

Reply:
OK. I'll bind with ManyToOne